tedana/gscontrol.py

```diff
@@ -38,93 +45,106 @@ def gscontrol_raw(catd, optcom, n_echos, io_generator, dtrank=4):
 
     Returns
     -------
-    dm_catd : (S x E x T) array_like
-        Input `catd` with global signal removed from time series
-    dm_optcom : (S x T) array_like
-        Input `optcom` with global signal removed from time series
+    data_cat_nogs : (S x E x T) array_like
+        Input ``data_cat`` with global signal removed from time series.
+    data_optcom_nogs : (S x T) array_like
+        Input ``data_optcom`` with global signal removed from time series.
     """
     LGR.info("Applying amplitude-based T1 equilibration correction")
```

Comment:
Is this accurate?

Reply:
I didn't write the original function, but this looks fine to me.